torch/csrc/jit/codegen/cuda/expr_simplifier.cpp

```diff
@@ -50,7 +51,13 @@ class Logger : public NoOpLogger {
     if (current_val_->sameAs(init_val_)) {
       return false;
     }
-    auto triggers = getDebugDumpArguments(DebugDumpOption::ExprSimplification);
+    const auto& triggers_arg =
+        getDebugDumpArguments(DebugDumpOption::ExprSimplification);
+    std::vector<std::regex> triggers;
```

Comment:
Using regex is convenient. Because I don't want to type the entire pass name, I could just do
```
$PYTORCH_NVFUSER_DUMP="expr_simplify(.*flatten)"
```